dev/README_RELEASE_AIRFLOW.md

```diff
@@ -182,6 +183,51 @@ https://pypi.python.org/pypi/apache-airflow
 It is important to stress that this snapshot should not be named "release", and it
 is not supposed to be used by and advertised to the end-users who do not read the devlist.
 
+- Push Tag for the release candidate
+
+This step should only be done now and not before, because it triggers an automated build of
+the production docker image, using the packages that are currently released in PyPI
+(both airflow and latest provider packages).
+
+```shell script
+git push origin ${VERSION}
+```
```

Comment:
```suggestion
    This step should only be done now and not before, because it triggers an automated build of
    the production docker image, using the packages that are currently released in PyPI
    (both airflow and latest provider packages).

    ```shell script
    git push origin ${VERSION}
    ```
```